cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -1162,13 +1171,30 @@ class SimpleArray
     {
         if (!m_shape.empty())
         {
-            m_buffer = buffer_type::construct(m_shape[0] * m_stride[0] * ITEMSIZE);
+            m_buffer = buffer_type::construct(m_shape[0] * m_stride[0] * ITEMSIZE, 0);
             m_body = m_buffer->template data<T>();
         }
     }
 
     // NOLINTNEXTLINE(modernize-pass-by-value)
-    explicit SimpleArray(small_vector<size_t> const & shape, value_type const & value)
+    SimpleArray(small_vector<size_t> const & shape, size_t alignment, with_alignment_t const & /* unnamed argument for tagging */)
```

Comment:
The comment `/* unnamed argument for tagging */` is redundant and should be removed.  Unnamed argument for tagging is a well-established convention that explains itself, just like SFINAE.